playground/mocks/data/idp/idx/error-identify-multiple-errors.json

```diff
@@ -0,0 +1,103 @@
+{
+  "version": "1.0.0",
+  "stateHandle": "02f9EONFMnQZtAQHl7Gf4Ye-R4mXc8cOhBUSMc7ubQ",
+  "expiresAt": "2021-11-02T22:10:41.000Z",
+  "step":"IDENTIFY",
+  "intent":"LOGIN",
+  "remediation":{
```

Comment:
i would also add another file that shows terminal view with multiple error messages.. meaning that it doesn't have `remediation` object instead only have `messages`